src/grid/grid.directive.ts

```diff
@@ -0,0 +1,53 @@
+import { Directive, HostBinding, Input, OnInit } from "@angular/core";
+
+@Directive({
+	selector: "[ibmGrid]"
+})
+export class GridDirective {
+	@HostBinding("class.bx--grid") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmRow]"
+})
+export class RowDirective {
+	@HostBinding("class.bx--row") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmCol]"
+})
+export class ColumnDirective implements OnInit {
+	@Input() class = "";
```

Comment:
How does this work with `columnClasses` host binding?
My impression would be that all classes would be accessible there and this would just be an input that has name collision with html keyword.

Reply:
dynamically added classes don't seem to be  available in host binding for directives